spec/models/program_spec.rb

```diff
@@ -90,7 +90,7 @@
 
   describe 'voting_period?' do
     it 'retuns false when voting dates are not set' do
-      expect(program.voting_period?).to eq false
+      expect(program.voting_period?).to eq true
```

Comment:
Until we did not have voting dates, the voting period was always on (technically even after the conference). I want to maintain that behavior for past conferences.